src/spring-cloud/azext_spring_cloud/tanzu.py

```diff
@@ -235,6 +239,46 @@ def tanzu_app_deploy(cmd, client, resource_group, service, name, artifact_path=N
     return client.deployments.deploy(resource_group, service, name, deployment_name, build_result_id=build_result_id)
 
 
+def tanzu_app_tail_log(cmd, client, resource_group, service, name, deployment=None, instance=None,

```

Comment:
See previous command, the deployment parameter is never used, let's remove it,

Reply:
Have removed the deployment parameter.